CMakeLists.txt

```diff
@@ -124,18 +124,20 @@ if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
     set(CMAKE_INSTALL_PREFIX "${PROJECT_BINARY_DIR}/dist" CACHE PATH ${INSTALL_PATH_DOC_STRING} FORCE)
 endif()
 
-set(HIPIFY_INSTALL_PATH ${CMAKE_INSTALL_PREFIX})
+set(HIPIFY_BIN_INSTALL_PATH "${CMAKE_INSTALL_PREFIX}/bin")
 
-install(TARGETS hipify-clang DESTINATION ${CMAKE_INSTALL_PREFIX}/bin)
+install(TARGETS hipify-clang DESTINATION bin)
```

Comment:
Shouldn't it be 
```cmake
install(TARGETS hipify-clang DESTINATION $(HIPIFY_BIN_INSTALL_PATH))
```
instead?

Reply:
By default ${CMAKE_INSTALL_PREFIX} will be added to install command DESTINATION, if the path is relative(ex: bin)
Better to use relative path in destination , rather than absolute path
https://cmake.org/cmake/help/latest/command/install.html